wagtail_localize/tests/test_edit_translation.py

```diff
@@ -2128,7 +2128,7 @@ def test_publish_snippet_translation(self):
         log = TranslationLog.objects.get()
         self.assertEqual(log.source, self.snippet_source)
         self.assertEqual(log.locale, self.fr_locale)
-        self.assertIsNone(log.page_revision)
+        self.assertIsInstance(log.page_revision, Revision)
```

Comment:
note to self: this is correct (in that TranslationLog does have a `page_revision` FK - https://github.com/wagtail/wagtail-localize/blob/a3efa8f65de3445c75ea271b1b0ee513a445deaa/wagtail_localize/models.py#L1368 - this used to point to PageRevision, but that changed. One for me to address

Reply:
Having said that, we do need a test for `TestSnippet` and drafts / publishing

Reply:
I've added a test to ensure draft state Snippets can be published in the same way as Pages (by `Translation.save_target()`)